README.md

```diff
@@ -113,7 +113,8 @@ The [**Google Sheets Setup & Authentication**](https://drive.google.com/open?id=
         "spreadsheet_id": "YOUR_GOOGLE_SPREADSHEET_ID",
         "start_date": "2019-01-01T00:00:00Z",
         "user_agent": "tap-google-sheets <api_user_email@example.com>",
-        "request_timeout": 300
+        "request_timeout": 300,
+        "supportsAllDrives": false
```

Comment:
The new config parameter should be in the snake case. `supports_all_drives`

Reply:
Updated configurable name as per suggetions